src/main/java/redis/clients/jedis/JedisClusterCommand.java

```diff
@@ -132,14 +159,60 @@ private T runWithRetries(final int slot) {
     throw maxAttemptsException;
```

Comment:
If `maxAttempts = 0`, we will get following exception():
```
Exception in thread "main" java.lang.NullPointerException: Cannot suppress a null exception.
	at java.base/java.lang.Throwable.addSuppressed(Throwable.java:1057)
	at redis.clients.jedis.JedisClusterCommand.runWithRetries(JedisClusterCommand.java:158)
	at redis.clients.jedis.JedisClusterCommand.run(JedisClusterCommand.java:45)
	at redis.clients.jedis.JedisCluster.set(JedisCluster.java:292)
	at redis.clients.jedis.tests.FancheTest.main(FancheTest.java:27)
```

but in the old code, we will get ` No more cluster attempts left.`, should we define default `lastException = maxAttemptsException`?

Reply:
@yangbodong22011 
1. This is the current behavior at `master` branch.
2. Why would you send `maxAttempts = 0`?

Reply:
> @yangbodong22011
> 
> 1. This is the current behavior at `master` branch.
> 2. Why would you send `maxAttempts = 0`?

1. ok, ignore that exception
2. just a test, or we should disable user Settings `maxAttempts = 0` (params check may be another PR), this PR can continue.